### DISCUSSION TITLE
Change order of pages

### DISCUSSION THREAD

**Discussion by denysbulavka** [xournalpp/xournalpp#2549] [General]
Hello,

I'm wondering, is it possible change the order of pages? Sometimes I create a new page in the wrong place and realize it later, so I would like to move this new page to the bottom, or to another place. Is this possible?

best,
denys.

**Comment by Technius** [comment]
You can move it by selecting the page in the sidebar and then clicking the up/down arrows to swap it with the previous or next page:
![image](https://user-images.githubusercontent.com/1066652/102560833-0f382600-4088-11eb-94c8-1d2560438575.png)

Unfortunately, I think it's currently only possible to swap with next/previous. Adding some controls to move it directly before/after some page N shouldn't be too hard, though (please submit an issue!).

**Reply by LittleHuba** [reply]
I'd like drag-and-drop + clipboard. No additional dialog and all functionality you need.

**Comment by v2belleville** [comment]
hi, I can not get the up down arrow to work for this. Eventually figured a right click opens a menu with move up / down item
<img src="https://user-images.githubusercontent.com/8779004/217811398-1c461122-b29c-48db-9db4-4d69773de589.png" width="400">

a dragNdrop feature would be great indeed !

**Comment by kalsifer-742** [comment]
Also the possibility to move multiple pages at once is missing
